docs/cloud/automation/terraform.md

```diff
@@ -1,2 +1,244 @@
+| network_id       | string | *Required*, the ID of the EventStore Cloud network into which the managed cluster should be created.                                                           | 
+| topology         | string | *Required*, the topology of the managed cluster. This determines the fault tolerance of the cluster. Valid values are `single-node` and `three-node-multi-az`. | 
+| instance_type    | string | *Required*, the size of the instances to use in the managed cluster                                                                                            | 
+| disk_size        | int    | *Required*, the size of the data disks in gigabytes                                                                                                            | 
+| disk_type        | string | *Required*, `GP2` (AWS), `premium-ssd-lrs` (Azure), `ssd` (GCP)                                                                                                | 
+| server_version   | string | *Required*,  `20.6`,`20.10`                                                                                                                                    | 
+| projection_level | string | *Optional*, default: `off`, the mode in which to enable projections. Valid values are `off`, `system`,  `user`                                                 | 
+
+
+::: tip
+ The actual implementation of each topology is specific to the resource provider.
+ 
+`instance_type` values can be `F1`, `C4`, `M8`, `M16`, `M32`, `M64`, `M128`  
+`disk_size`:  The minimum size is 10GB.  
+`disk_type`: The values depend on the cloud you're using `GP2` (AWS), `premium-ssd-lrs` (Azure), `ssd` (GCP)
+::: 
+
+### Attributes
+
+| Name              | Type   | Description                                                                                    | 
+| :---------        | :----- | :-------------                                                                                 | 
+| id                | string | the ID of the cluster.                                                                         | 
+| dns_name          | string | the DNS name at which the cluster can be found                                                 | 
+| resource_provider | string | the resource provider into which the cluster was provisioned.                                  | 
+| region            | string | the region in which the cluster was provisioned.                                               | 
+| gcp_network_name  | string | network name for the peering link in GCP                                                       | 
+| gcp_network_id    | string | GCP Network ID in URL format which can be passed to `google_compute_network_peering` resources | 
+
+
+::: tip
+`region` and `resource_provider` values are controlled by the network in which the cluster is created.
+:::
+
+### Creating a cluster (AWS)
```

Comment:
**SUGGESTION:** Could we add the code switch with the full sample configuration also for Azure and GCP?